Mupa1/StackOverFlowLite1
# Ft login page 162590072 UI

__What does this PR do__
Adds login page for the user

__Description of task to be completed__
Create a log in page with the following features:-
    1. A descriptive login form
    2. Link to sign up 

__How should this be manually tested__
    1. Clone this repository
    2. cd into the StackOverFlowLite1 folder
    3. Open login.html in your browser

__What are the relevant pivotal tracker stories__
#162638962 A User should be able to see the landing page

Screenshot
![login_page](https://user-images.githubusercontent.com/44979953/50042404-4fb4d180-0073-11e9-81c1-f962a0caa78f.png)
